app/controllers/ems_common.rb

```diff
@@ -746,6 +746,14 @@ def form_instance_vars
     @vmware_cloud_api_versions = retrieve_vmware_cloud_api_versions
     @emstype_display = model.supported_types_and_descriptions_hash[@ems.emstype]
     @nuage_api_versions = retrieve_nuage_api_versions
+    @hawkular_security_protocols = retrieve_hawkular_security_protocols
+  end
+
+  def retrieve_hawkular_security_protocols
+    [[_('SSL'), 'ssl-with-validation'],
+     [_('SSL trusting custom CA'), 'ssl-with-validation-custom-ca'],
+     [_('SSL without validation'), 'ssl-without-validation'],
+     [_('Non-SSL'), 'non-ssl']]
```

Comment:
Looks like the removed code was added back.
Why was it removed though? Was it removed inadvertently?